src/styles/components/_topic.scss

```diff
@@ -0,0 +1,58 @@
+@import '../tools/breakpoints';
+@import '../tools/mixins';
+@import '../settings/colors';
+
+// =============================================================================
+// TOPIC OVERVIEW
+//
+// Topics are the sections of a collection and can be broken into subtopics.
+//
+// =============================================================================
+
+$TOPIC_SPACING_SIZE_LARGE: 48px;
+$TOPIC_SPACING_SIZE_MEDIUM: 40px;
+$TOPIC_SPACING_SIZE_SMALL: 32px;
+
+.w-topics {
+  .w-topic {
```

Comment:
I think we could BEM these up a bit so they don't rely on descendant selectors. As much as possible we try to avoid descendant selectors to avoid specificity weirdness.

Maybe something like:

```
.w-topic {...}
.w-topic__header {...}
.w-topic__subtopic-header {...}
```